src/modules/helpchan.ts

```diff
@@ -177,10 +177,17 @@ export class HelpChanModule extends Module {
 			if (dormant && dormant instanceof TextChannel) {
 				await this.moveChannel(dormant, categories.ask);
 
-				const lastMessage = dormant.messages.cache
+				let lastMessage = dormant.messages.cache
 					.array()
 					.reverse()
 					.find(m => m.author.id === this.client.user?.id);
+
+				if (!lastMessage)
+					lastMessage = (await dormant.messages.fetch({ limit: 5 }))
```

Comment:
@ronthecookie nope. Actually, 1 would be fine. This is to prevent situations where the "this channel is now dormant" is not in it's cache as it's been restarted. Because the channel is locked, there is no possible way there can be more then 1, although there is the off chance an admin sent something, so keeping it at 5 is good imo. 

Resolving this.